README.md

```diff
@@ -47,14 +47,14 @@ In this section we'll use the Azure Portal to create the required resources to h
 
 1. Ensure the _Subscription_ and _Resource Group_ are correct, then provide the following configuration for the storage account:
 
-   - _Name_ - `my-keystone-app`
```

Comment:
Can’t use hyphens, only for storage accounts -

“The field can contain only lowercase letters and numbers. Name must be between 3 and 24 characters.”

Reply:
🤦 of course, I always stuff that up